src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java

```diff
@@ -90,6 +91,7 @@
     private static final String WEBDRIVER_BROWSER_VERSION = Props.get("webdriver.browser.version");
     private static final String WEBDRIVER_OS_ARCHITECTURE = Props.get("webdriver.os.arch");
     private static final String WEBDRIVER_BROWSER_PATH = Props.get("webdriver.browser.path");
+    private static final String WEBDRIVER_BROWSER_SIZE = Props.get("webdriver.browser.size");
```

Comment:
Please add info about this param to https://github.com/sbtqa/page-factory/blob/master/src/main/resources/config/application.properties-sample

Reply:
Done